Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -392,7 +392,11 @@ void GameList::ShowContextMenu(const QPoint&)
 
 void GameList::OpenProperties()
 {
-  PropertiesDialog* properties = new PropertiesDialog(this, *GetSelectedGame());
+  const std::shared_ptr<const UICommon::GameFile> game = GetSelectedGame();
```

Comment:
Any particular reason why you type out the full type name but use `auto` in all the other methods?
For consistency, you should probably be using `auto` here as well (not to mention because the type name doesn't really add too much value being here)

Reply:
Changed to `auto` as requested for consistency.